hopp/simulation/technologies/sites/site_info.py

```diff
@@ -138,7 +138,7 @@ def __attrs_post_init__(self):
         self.elec_prices = ElectricityPrices(data['lat'], data['lon'], data['year'], filepath=self.grid_resource_file)
         self.n_periods_per_day = self.n_timesteps // 365  # TODO: Does not handle leap years well
         self.interval = int((60*24)/self.n_periods_per_day)
-        self.urdb_label = data['urdb_label'] if 'urdb_label' in data.keys() else None
+        self.urdb_label = self.data['urdb_label'] if 'urdb_label' in self.data.keys() else None
```

Comment:
we shouldn't need this change due to L105. Though if we want we can simply have everything use `self.data` instead of local variable `data = self.data`